api/routing/routes.go

```diff
@@ -111,8 +80,17 @@ func NewRouter(tm db.TransactionManager, version, hateoasBaseBath, uiBasePath st
 	unsecured.GET("/openapi.json", nil, router.OpenAPI(infos, "json"))
 	unsecured.GET("/openapi.yaml", nil, router.OpenAPI(infos, "yaml"))
 
-	// redirect unknown routes to angular
-	router.Engine().NoRoute(uiRedirectHandler(uiBasePath))
+	if _, err := os.Stat(webUIDir + "/index.html"); os.IsNotExist(err) {
+		log.Warn("index.html not found. Starting in API only mode. Nop static content will be served.")
```

Comment:
`s/Nop/No`

Reply:
add `return router` here and remove the `else {` to keep a single code path

Reply:
Done